Assembler.Core/RawAssemblingEntities/RawMessageInAssembly.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Collections.Generic;
+using Assembler.Core.Entities;
+
+namespace Assembler.Core.RawAssemblingEntities
+{
+    public class RawMessageInAssembly : BaseMessageInAssembly
+    {
+        public List<BaseFrame> AssembledFrames { get; }
```

Comment:
Why not have 3 lists - InitialFrames, MiddleFrames and EndFrames?

Reply:
I thought about it, and I don't see the advantage of it, you probably would need another resolver anyway, and that way you have to configure 1 instead  of 3.